packages/cli/src/lib/snapshot/snapshotConstant.ts

```diff
@@ -0,0 +1,29 @@
+import {ResourceSnapshotType} from '@coveord/platform-client';
```

Comment:
mmh.. good question. It definitively should not be in the CLI, but not sure the platform-client is the right place.
I pasted the dict here since, in our last discussion, we said we would simply go with a dummy copy paste for now.

Reply:
Yep as @y-lakhdar said, we're not sure yet where the common idiom between the Admin Console and the CLI should reside.
It's technically not the job of the platform-client to bother about what's the good label for XYZ.
So, for the time being, copy-pasta it is.